app.py

```diff
@@ -84,46 +84,46 @@ def upload():
 
 
 	file_uploaded.save(path)  # save to disk
-	print '%s: saved to disk' % uuid
+	print ('%s: saved to disk') % uuid
```

Comment:
`%` should be inside the parenthesis. Probably better to move to `{}` [formatting](https://pyformat.info/).